### ISSUE TITLE
No need to generate gql-tag object, then print it to graphql-request 

### ISSUE THREAD

**Issue by lemiesz** [dotansimha/graphql-code-generator#2854]
I have a query file like this 

```
getRocketQuery.graphql
query getRocket($limit: Int!) {
  launchesPast(limit: $limit) {
    mission_name
  }
}
```

I am running with the following plugins

```
  src/generated/graphql.ts:
    plugins:
      - "typescript"
  src/generated/operations.ts:
    plugins:
      - "typescript-operations"
      - "typescript-graphql-request"
```

what gets generated is something like this 
```
export const GetRocketDocument = gql`
  query getRocket($limit: Int!) {
    launchesPast(limit: $limit) {
      mission_name
    }
  }

export function getSdk(client: GraphQLClient) {
  return {
    getRocket(variables: GetRocketQueryVariables): Promise<GetRocketQuery> {
      return client.request<GetRocketQuery>(
        print(GetRocketDocument),
        variables
      );
    }
  };
}
`;
```

It seems strange that now I require two more libraries in my runtime bundle `gql` and `print from graphql`. In addition it seems like the client needs to generate an gql AST just to convert back to string at runtime. 

Could this template just bypass all of this, and just create the graphql string at runtime.

**Comment by dotansimha** [comment]
@lemiesz I added an additional configuration for `documentMode`, now it accepts `string` option. 

Use:
```yml
generates:
  src/generated/sdk.ts:
    config:
      documentMode: 'string'
    plugins:
      - typescript
      - typescript-operations
      - typescript-graphql-request
```

You should use the latest alpha (`1.8.3-alpha-d0acf100.27`), since it's not in a release yet.

**Comment by lemiesz** [comment]
Cool thanksalot for this man

**Comment by ardatan** [comment]
Let's keep this open until it is released

**Comment by dotansimha** [comment]
Fixed in 1.8.3
